lib/backend/k8s/resources/names.go

```diff
@@ -28,8 +28,29 @@ import (
 
 // IPToResourceName converts an IP address to a name used for a k8s resource.
 func IPToResourceName(ip net.IP) string {
```

Comment:
I think there are some corresponding changes / cleanup we can do here: https://github.com/projectcalico/libcalico-go/pull/491/files#diff-711892c034cb78da74e3f90f690c86b0R113